ci/scripts/driver_weekly.sh

```diff
@@ -22,7 +22,7 @@ set -eux
 # TODO using static build for GitHub CLI until fixed in HPC-Stack
 #################################################################
 export GH=${HOME}/bin/gh
-export REPO_URL=${REPO_URL:-"https://github.com/NOAA-EMC/global-workflow.git"}
+export REPO_URL="ssh://git@ssh.github.com:443/NOAA-EMC/global-workflow.git"
```

Comment:
Can I ask why `ssh`?

Reply:
more secure

Reply:
I'm still nervous about making/deleting branches in the auth repo. But I agree to not having these in a personal account either. Can emc-bot make a clone of global-workflow where the temp branches could be made?